docs/en/observability/logs-troubleshooting.asciidoc

```diff
@@ -1,4 +1,111 @@
 [[logs-troubleshooting]]
 = Troubleshoot your logs
 
-_coming soon_
\ No newline at end of file
+This section provides solutions to errors you might encounter while onboarding your logs in {kib}.
```

Comment:
I stumbled over `in {kib}`. Do you onboard your logs in Kibana? To Elasticsearch? To the Elastic Stack? How does a new user think here?

Reply:
My thought was that these current troubleshooting steps are mostly for errors users encounter while using the Kibana onboarding, so I was trying to be more specific there. Maybe even the title of this doc can be Logs onboarding troubleshooting until it is filled out with more generic troubleshooting information.